torch/lib/THD/base/init_methods/InitMethodUtils.cpp

```diff
@@ -83,4 +83,28 @@ std::pair<std::string, std::string> discoverMaster(std::vector<std::string> addr
   return std::make_pair(master_address, my_address);
 }
 
+// Get the rank based on the order of writing, helper for file:// init method
```

Comment:
If it's for `file://` only then why is it here? Do you need it for other init methods?

Reply:
This is needed for other method, that's why i refactored it out，removed the comment